package/MDAnalysis/coordinates/PDB.py

```diff
@@ -475,6 +475,8 @@ class PDBWriter(base.WriterBase):
     .. _MODEL: http://www.wwpdb.org/documentation/file-format-content/format32/sect9.html#MODEL
     .. _ENDMDL: http://www.wwpdb.org/documentation/file-format-content/format32/sect9.html#ENDMDL
     .. _CONECT: http://www.wwpdb.org/documentation/file-format-content/format32/sect10.html#CONECT
+    .. _ATOM: http://www.wwpdb.org/documentation/file-format-content/format32/sect9.html#ATOM
```

Comment:
As things stand currently, these links don't actually do anything so we should use them to add links in the docs (see below).

Reply:
Done